ingest/src/main/java/com/microsoft/azure/kusto/ingest/ResourceManager.java

```diff
@@ -346,16 +372,121 @@ protected String retrieveServiceType() {
         return null;
     }
 
+    @Override
+    public void reportIngestionResult(ResourceWithSas<?> resource, boolean success) {
+        if (storageAccountSet == null) {
+            log.warn("StorageAccountSet is null");
+        }
+        storageAccountSet.addResultToAccount(resource.getAccountName(), success);
+    }
+
+    private int calculateRetries(int size) {
```

Comment:
@yogilad  - i think we changed or are changing this in .Net - arent we?

Reply:
Yes. Should be default 3 with policy override

Reply:
Done